src/objective-c/GRPCClient/private/GRPCHost.m

```diff
@@ -78,6 +78,7 @@ - (instancetype)initWithAddress:(NSString *)address {
     if (cachedHost) {
       return cachedHost;
     }
+  }
```

Comment:
Argh, I saw the breakage and just typed the semicolon mindlessly. Thanks for catching it.